Supporting Resources/v2.0/Datamodel/Datafiles/Question D-SR-2-B.yml

```diff
@@ -16,18 +16,18 @@ id:
 
 #One-sentence description of the criterium
 text: 
-  Does the vendor meet the security responsibilities and quality measures to be in line with service level agreements as defined by the organization?
+  Do vendors meet the security responsibilities and quality measures of service level agreements defined by the organization?
 
 #Order of this question for the activity (in case there would be multiple)
 order:
   1
 
 #Qualifying Criterion
 quality:
-  - During the creation of vendor agreements, security requirements are discussed with the vendor.
-  - Vendor agreements provide specific guidance on security defect remediation within an agreed upon timeframe.
-  - The organization has a templated agreement of responsibilities and service levels for key vendor security processes.
-  - Key performance indicators are measured.
+  - You discuss security requirements with the vendor wehen creating vendor agreements
```

Comment:
can you fix the typo wehen?

Reply:
@Pat-Duarte updated phrasing makes sense to me, do you fix the typo? I will then merge. thx